tips_and_tricks/tips_and_tricks.md

```diff
@@ -72,7 +72,7 @@ Click on the **New** button and add/paste the link to your repository in the app
 Also you can use the terminal to commit and push changes to your repository in GitHub.
 
 ## Time Travel Diffs
-The editor based documents (e.g. python code, sage worksheets, LaTex documents, etc.) as well as the Jupyter notebooks and Sage worksheets are Time Travel Diffs supported. The Time Travel Diffs feature allows you to see what happened with a file within a certain time interval.
+The editor based documents (e.g. python code, LaTeX documents, markdown files, etc.) as well as Jupyter notebooks and Sage worksheets are Time Travel Diffs supported. The Time Travel Diffs feature allows you to see what happened with a file within a certain time interval.
```

Comment:
python --> Python

Reply:
Markdown instead of markdown?  I'm not sure.

Reply:
"are Time Travel Diffs supported." --> "support using TimeTravel diffs".  

I think all doctypes support diffs by the way.  And it's "TimeTravel", not "Time Travel".